frontend/src/components/notifications/Notifications.tsx

```diff
@@ -1,18 +1,54 @@
 import { ReactNode } from "react";
 import { Id, toast, ToastContainer, ToastOptions, TypeOptions } from "react-toastify";
+import { faCopy, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+import { twMerge } from "tailwind-merge";
+
+import { CopyButton } from "../v2/CopyButton";
 
 export type TNotification = {
   title?: string;
   text: ReactNode;
   children?: ReactNode;
+  cta?: ReactNode;
```

Comment:
Nit: I was confused what this actually meant, can we rename it to `callToAction`?